webpack/redux/actions/RedHatRepositories/repositorySetRepositories.js

```diff
@@ -68,13 +68,8 @@ const loadRepositorySetRepos = (contentId, productId) => async (dispatch) => {
       productId,
       results: data.results,
     });
-  } catch (response) {
-    const error = response && response.data && response.data.error;
-    return dispatch({
-      type: REPOSITORY_SET_REPOSITORIES_FAILURE,
-      contentId,
-      error,
-    });
+  } catch (error) {
+    return dispatch(apiError(REPOSITORY_SET_REPOSITORIES_FAILURE, error, contentId));
```

Comment:
This seems to end the loading state correctly..
```suggestion
    return dispatch(apiError(REPOSITORY_SET_REPOSITORIES_FAILURE, error, { contentId: contentId }));
```

Reply:
Ok, thank you. Let me run the tests and see if it plays nicely with the change in the reducer.